frontend/src/store/store.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 import createPersistedState from 'vuex-persistedstate'
-import VueJwtDecode from 'vue-jwt-decode'
+// import VueJwtDecode from 'vue-jwt-decode'
```

Comment:
please remove and also remove the package

Reply:
Yes, I kept it there because maybe we want to decode when the token expires. But we could also set this in the config.